reportportal/templates/service-api/api-deployment.yaml

```diff
@@ -20,7 +20,7 @@ spec:
         {{ $key }}: {{ $value | quote }}
         {{- end }}
     spec:
-      {{- with .Values.imagePullSecrets }}
+      {{- if .Values.imagePullSecrets }}
```

Comment:
Please use `with` instead of `if` https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with

Reply:
Adjusted to use this instead and pushed changes here